src/PokemonTrainer.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class PokemonTrainer {
+    private String userA;
+    private List<Pokemon> pokemons = Arrays.asList();
+
+
+    public PokemonTrainer(String userA, List<Pokemon> pokemons) {
+        this.userA = userA;
+        this.pokemons = pokemons;
+    }
+
+    public String getName() {
```

Comment:
De getName methode returned meestal een "name" variabele. Jij hebt het nu "userA" genoemd, maar dat is geen goede naam.